notebooks/02_foundations__connection_riemannian_metric.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p>NIT: quotes in latex: <code></code>g"</p><p> </p><p>NIT: Christoffel symbols \Gamma^k_{ij} (without the \partial_k)</p><p>NIT: christoffel -&gt; Christoffel</p><p> </p><p>NIT: However, we can only calculate these christoffel symbols if we have equip the manifold with a metric.</p><p>--&gt; In the case of a Levi-Civita connection, the Christoffel symbols can be calculated from the metric as follows: ... &lt;-- and remove \partial_k in the left of the equation</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='90'/>

Reply:
<p>:D</p>